src/tests/historical_balance_test.ts

```diff
@@ -0,0 +1,31 @@
+import { HistoricalBalance } from "../coinbase/historical_balance";
+import { HistoricalBalance as HistoricalBalanceModel } from "../client";
+import { Decimal } from "decimal.js";
+import { Coinbase } from "../coinbase/coinbase";
+
+describe("Balance", () => {
+  describe(".fromModel", () => {
```

Comment:
```suggestion
  describe("#fromModel", () => {
```

Please prefix static methods with `#methodName` for jest descriptions